CodeEditModules/Modules/StatusBar/src/StatusBar.swift

```diff
@@ -8,6 +8,13 @@
 import SwiftUI
 import GitClient
 
+func checkIfHovering(isHovering: Bool) {
```

Comment:
Global functions are actually not a good idea.

Also the argument label in this case is redundant. The function already says it.

Reply:
I would suggest:

```swift
func isHovering(_ active: Bool) {
    if active {
        NSCursor.pointingHand.push()
    } else {
        NSCursor.pop()
    }
}
```